### DISCUSSION TITLE
How leave the invited repo?

### DISCUSSION THREAD

**Discussion by malrangcow00** [community/community#67217] [New to GitHub] [answered]
repo owner already sent me out 
but repo is still on my left side menu

**Comment by elendil7** [comment]
According to [GitHub's documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/removing-yourself-from-a-collaborators-repository), this can be done as follows:

1. In the upper-right corner of any page on github.com, click your profile photo, then click Settings.
2. In the "Code, planning, and automation" section of the sidebar, click **Repositories**.
3. Next to the repository you want to leave, click **Leave**.
4. Read the warning carefully, then click **I understand**, to leave this repository.

**Reply by malrangcow00** [reply]
I can't see that repo on that menu but still exist on my left side bar

**Reply by elendil7** [reply]
If you're 100% sure you've left the repository:

Perhaps clearing your browser cache could help? Depending on browser, you can press CTRL+ALT+DEL to bring up the deletion window, and make sure "cache" is selected before deleting.

Alternatively, try logging out then back into your github account.

**Reply by malrangcow00** [reply]
![image](https://github.com/community/community/assets/115908997/06f68ff6-aa10-4d12-9d02-7b7a19f08c87)
after delete cache it appeared but there's no leave button here

**Reply by elendil7** [reply]
Ah, it looks like you are the **owner** of the repository. Your friend has "_forked_" it, meaning they have saved a copy of it on their GitHub account (to continue development on by themselves).

Since there is no way to leave a repository you own, your only choice would be to **delete** it.

So you can feel free to delete the repository, if you no longer wish to work on it. See link below:

https://docs.github.com/en/repositories/creating-and-managing-repositories/deleting-a-repository

**Reply by malrangcow00** [reply]
I really appreciate you
![image](https://github.com/community/community/assets/115908997/606b2fc0-666b-46fc-95ca-799b0999d490)
Despite deleting the forked repo, the original remains and is not visible in my repository
but still on my left ...
![image](https://github.com/community/community/assets/115908997/d4f0efbf-acc3-4fe5-8dc9-0b235c8d6013)

**Reply by elendil7** [reply, marked as answer]
Perhaps try this?

https://github.com/gogs/gogs/issues/535#issuecomment-678685869

"```I discovered the solution. After deleting, you must go back into the repo, select Settings > Manage Access > Manage, then delete again, using the same process.```"

Also, check out the following post which explains that this issue may be related to caching:

https://github.com/desktop/desktop/issues/8586

Steps you could take:
- Delete browser cache again
- Restart the GitHub Desktop app (if you have it)
- Restart your laptop / computer
- Try logging into GitHub in a private browser window, and see if the issue persists.

**Reply by malrangcow00** [reply]
Thank you for your help !
finally I blocked my friends lol ..
and it disappeared
